spec/classes/agent_spec.rb

```diff
@@ -98,7 +174,11 @@
         context "with backends => #{backends}" do
           let(:params) { { :backends => backends } }
 
-          it { should compile.with_all_deps }
+          # Unknown bug in rspec-puppet fails to compile windows paths
```

Comment:
could you please open a ticket with rspec-puppet about this and put the issue URL in the comment

Reply:
Updated comments to mention https://github.com/rodjek/rspec-puppet/issues/750